### ISSUE TITLE
implementation details about textmodel_svm ?

### ISSUE THREAD

**Issue by randomgambit** [quanteda/quanteda.textmodels#47]
Hello there!

I hope all is well during these difficult times! I was playing with the great `quanteda` and discovered the nice `textmodel_svm` classification model. However, contrary to `textmodel_nb` where there is a little example which reproduces juravsky's toy case, I cannot find anything about `textmodel_svm`. 

Are any additional details available about this function (a quanteda tutorial, a toy example, etc)? What is happening under the hood when using `textmodel_svm` with `dfm`s? Can we get back the coefficients for each token?

Thanks!

**Comment by kbenoit** [comment]
Yes we realised this recently... See #45. Easily overridden through `type`, which is passed via `...`.

Documentation is available in the references to `textmodel_svm()` as to how this works.

**Comment by kbenoit** [comment]
I think they came from running the code outside of the quanteda structure, so we are verifying it against running a non-quanteda version of the model with the same model data. Not a very strong test, but does check whether something went amiss in our wrapper.

Would be delighted for more critical tests or feedback, if you have it.

**Comment by randomgambit** [comment]
I am looking for some interesting docs. By the way, out of curiosity, do you know how does `predict_svm` recover the predicted probabilities for instance when using penalized logistic classification? Given K classes, is the algorithm fitting K one-vs-all classifiers, computing all K probabilities and then normalizing each probability by the sum of these probabilities (so that the sum is indeed one)? What do you think?

**Comment by kbenoit** [comment]
That's in the paper describing the method, but for multinomial logistic regression (of which the penalised approach is a special version), these are equivalent. The standard way is to compute this as per the last equation in https://en.wikipedia.org/wiki/Multinomial_logistic_regression#As_a_set_of_independent_binary_regressions.
